app/src/androidTest/java/com/github/factotum_sdp/factotum/ui/directory/ContactsTestHelperMethods.kt

```diff
@@ -0,0 +1,7 @@
+package com.github.factotum_sdp.factotum.ui.directory
+
+import com.google.firebase.database.FirebaseDatabase
+
+fun emptyFirebaseDatabase(database: FirebaseDatabase) {
```

Comment:
This should be in a utils directory in a proper class.

Reply:
I don't really understand where you would put it, should I create a folder utils in my directory folder in the tests or a general utils folder for all tests. I don't really see the added benefit of putting it inside a class though.

Reply:
I think a general utils for all tests will be good.
The class would be ContactsUtils.
One class for every test type that needs it.